plugins/services/src/js/components/modals/ServiceDestroyModal.js

```diff
@@ -140,43 +132,144 @@ class ServiceDestroyModal extends React.Component {
     }, REDIRECT_DELAY);
   }
 
-  render() {
-    const { isPending, onClose, open, service } = this.props;
+  getCloseButton() {
+    const { onClose } = this.props;
+
+    return (
+      <div className="row text-align-center">
+        <button
+          className="button button-primary button-medium"
+          onClick={onClose}
+        >
+          Close
+        </button>
+      </div>
+    );
+  }
+
+  getDestroyFrameworkModal() {
+    const { open, service, onClose, intl } = this.props;
+    const serviceName = service ? service.getId() : "";
+
+    return (
+      <Modal
+        header={this.getModalHeading()}
+        footer={this.getCloseButton()}
+        modalClass="modal"
+        onClose={onClose}
+        open={open}
+        showHeader={true}
+        showFooter={true}
+        subHeader={this.getSubHeader()}
+      >
+        <p>
+          {intl.formatMessage({
```

Comment:
Because the the modals are not being rendered inside the main application in which `IntlProvider` is being used. This is a "workaround" we started using while we don't refactor the portals to be rendered part of the main application.